test/e2e-test.mk

```diff
@@ -42,11 +45,6 @@ clean-spoke: clean-spoke-cr clean-spoke-operator
 clean-e2e:
 	$(RM) ./e2e.test
 
-cluster-ip:
```

Comment:
The only usage of `cluster-ip` is before `run-e2e`. 
And the only usage of `run-e2e` is the last step of `test-e2e`.

That's mean all HUB_KUBECONFIG must be processed by `cluster-ip` in our e2e. That's the reason I conbine these 2 commands into 1.

And I also move `bootstrap-secret` to `test-e2e` to make `run-e2e` more indenpendent. The reason is test-e2e represents a default/standard way of preparing e2e testing env.   But for cases that we want to do some motifications, we can reuse the `run-e2e`.